### DISCUSSION TITLE
How to get a constant latency value for audio recordings using oboe library?

### DISCUSSION THREAD

**Discussion by saranshg20** [google/oboe#1556] [Q&A] [answered]
I am currently working on an android application that uses the oboe library for recording audio. The goal of the application is to get a constant audio latency. Is it possible to get a constant latency audio recording using the oboe library?

**Comment by robertwu1** [comment]
Yes, see the Record and Play in OboeTester as a sample.

**Comment by philburk** [comment, marked as answer]
Once a stream is running and stabilized then the latency will be relatively constant.
But if you start to starve an output stream then the output buffer will get low.
That will cause a temporary reduction in latency.

If you do not process an input stream in time then that will cause a temporary increase in latency.

This is unavoidable.
